### DISCUSSION TITLE
Plea to restore HC character

### DISCUSSION THREAD

**Discussion by Cavebunny** [chromiecraft/chromiecraft#6362] [General]
I'm afraid I did not record.
Kozi flew from Stormwind to Ironforge. Dead at IF flight master due to falling through "floor". Could you please restore my character?

![WoWScrnShot_112623_143929](https://github.com/chromiecraft/chromiecraft/assets/152076514/5e96a74b-930f-4965-a08c-6336ee88e7f0)
![WoWScrnShot_112623_143829](https://github.com/chromiecraft/chromiecraft/assets/152076514/f0c487b7-ac8a-49cb-9170-0c1ab8675796)
![WoWScrnShot_112623_143801](https://github.com/chromiecraft/chromiecraft/assets/152076514/cab1bad7-c3fc-41a7-83f3-99ab373389cf)
![WoWScrnShot_112623_143723](https://github.com/chromiecraft/chromiecraft/assets/152076514/3db9ab79-4f8b-4ee1-9e37-c2291ca3863c)
![WoWScrnShot_112623_143653](https://github.com/chromiecraft/chromiecraft/assets/152076514/3be2e849-8a67-4f40-9424-bbd4231b4ffb)
![WoWScrnShot_112623_143600](https://github.com/chromiecraft/chromiecraft/assets/152076514/9ea4e62d-d47b-4527-8d21-5db41f0ef84e)

**Comment by BeefSteaker** [comment]
Ive restored your character. If you have access, #support-ticket on the discord is a better way to contact us for issues like this

**Reply by Cavebunny** [reply]
Thanks a lot!
I did go there first, but actually do not have a phone. It's the honest truth; I've been trying to live without it for over five years now. Only using email. Of course I have access to phone at work, but I wouldn't use that to verify discord.
